openbas-api/src/main/java/io/openbas/migration/V4_33__Force_redindex_scenario.java

```diff
@@ -0,0 +1,18 @@
+package io.openbas.migration;
+
+import java.sql.Statement;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.springframework.stereotype.Component;
+
+@Component
+public class V4_33__Force_redindex_scenario extends BaseJavaMigration {
+  @Override
+  public void migrate(Context context) throws Exception {
+    try (Statement statement = context.getConnection().createStatement()) {
+      // re-index scenarios and inject-expectation
+      statement.executeUpdate(
+          "DELETE FROM indexing_status WHERE indexing_status_type in ( 'scenario', 'expectation-inject');");
+    }
+  }
+}
```

Comment:
@guillaumejparis & @Dimfacion we need to find a better way to reindex ES.
A config file with timestamp for example or something else, but write a migration for this seems overkill.